core/src/components/item/item.md.scss

```diff
@@ -471,3 +473,10 @@
     --border-color: #{$item-md-input-fill-border-color-hover};
   }
 }
+
+// Material Design Text Field Character Counter
+// --------------------------------------------------
+
+.item-counter {
+  letter-spacing: var(--counter-letter-spacing);
```

Comment:
Why are we adding this?

Reply:
In the Material Design [live preview](https://material.io/components/text-fields) if you enable character counter and inspect their CSS, they have letter spacing applied to the design.

Reply:
Ok cool 👍  Not sure we need this exposed as a public API for folks to control though. Thoughts?